src/presenter/cerebral/actions/resetFormAction.test.js

```diff
@@ -0,0 +1,20 @@
+import { defaultForm } from '../../defaultForm';
+import { runAction } from 'cerebral/test';
+
+import { resetFormAction } from './resetFormAction';
+
+describe('resetFormAction', () => {
+  it('should reset form data', async () => {
+    const initialState = {
+      state: {
+        form: {
+          ...defaultForm(),
+          city: 'Testing',
+        },
+      },
+    };
+    return runAction(resetFormAction, initialState).then(({ state }) =>
```

Comment:
I would recommend not mixing `async` with Promise calls like `.then`.  Rather, I would prefer to see 
`const { state } = await runAction(...);` followed by the assertion.

Reply:
this is what happens when I copy/paste from SO :P